app/src/main/java/com/infomaniak/mail/data/api/ApiRepository.kt

```diff
@@ -98,7 +99,8 @@ object ApiRepository : ApiRepositoryCore() {
     }
 
     fun setDefaultSignature(mailboxHostingId: Int, mailboxName: String, signature: Signature): ApiResponse<Boolean> {
```

Comment:
We should add the possibility to set the Signature to "no signature" in the Signature settings.